src/main/java/fr/openent/lystore/export/campaign/CampaignExport.java

```diff
@@ -23,21 +24,20 @@
 import java.util.List;
 import java.util.Map;
 
+import static fr.openent.lystore.constants.ParametersConstants.REGION_TYPE_NAME;
+
 public class CampaignExport extends ExportObject {
-    private JsonObject params;
-    private ExportService exportService;
-    private Logger log = LoggerFactory.getLogger(fr.openent.lystore.export.validOrders.ValidOrders.class);
-    private String idFile;
-    private String url;
-    private Integer id;
-    private List<Integer> ids;
-    private SupplierService supplierService;
+    private final Logger log = LoggerFactory.getLogger(fr.openent.lystore.export.validOrders.ValidOrders.class);
```

Comment:
Certainement une indentation mais si tu peux le corriger la signature de la classe de ton logger :

  ```suggestion
    private final Logger log = LoggerFactory.getLogger(CampaignExport.class);
```